Sources/Drain.swift

```diff
@@ -51,22 +49,15 @@ public final class Drain: DataRepresentable, Stream {
     }
 
     public func receive() throws -> Data {
-        let data = self.data
         close()
-        return data
+        return buffer
     }
 
     public func send(data: Data) throws {
-        enum Error: ErrorProtocol {
-            case sendUnsupported
-        }
-        throw Error.sendUnsupported
+        buffer += data.bytes
     }
```

Comment:
I think we can use `buffer.appendContents(of: data)`. I don't remember the correct syntax, but anyway, I think it's best to use the function instead of the operator.